edweiss-firebase/functions/src/functions/course/updateMaterial.ts

```diff
@@ -51,10 +51,13 @@ export const updateMaterial = onSanitizedCall(Functions.updateMaterial, {
         return fail("invalid_json");
     }
 
+    assertIsBetween(materialData.title.length, 1, MAX_MATERIAL_TITLE_LENGTH, "material_title_too_long");
+    assertIsBetween(materialData.description.length, 0, MAX_MATERIAL_DESCRIPTION_LENGTH, "material_description_too_long");
+
     // Prepare updated fields
     const updatedFields: Partial<Material> = {};
     if (materialData.title) updatedFields.title = materialData.title;
-    if (materialData.description) updatedFields.description = materialData.description;
+    if (materialData.description || materialData.description === '') updatedFields.description = materialData.description;
```

Comment:
I think I prefer use the current because `materialData.description !== undefined` does not test if it is null. However `materialData.description` checks that it is neither null nor undefined, nor empty nor 0.